cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -299,6 +299,19 @@
 
 # Valid var per schema
 good_var = pd.DataFrame(
+    [
+        [False],
+        [False],
+        [False],
+        [False],
+        [False],
+    ],
+    index=["ERCC-00002", "ENSG00000127603", "ENSMUSG00000059552", "ENSSASG00005000004", "FBtr0472816_df_nrg"],
+    columns=["feature_is_filtered"],
+)
+
+# Valid var per schema
+good_var_four_genes = pd.DataFrame(
```

Comment:
i added the fruit fly gene to `good_var`, and kept what we previously had as `good_var_four_genes` so that we wouldn't need to rewrite the matrix math in `test_determine_seurat_convertibility`

Reply:
do you mind leaving a comment linking this to seurat convertibility testing? so we know we can safely delete if we decide to merge this feature branch into main post seurat deprecation